ISHPullUpSample/ISHPullUpSample/BottomVC.swift

```diff
@@ -112,6 +113,8 @@ class BottomVC: UIViewController, ISHPullUpSizingDelegate, ISHPullUpStateDelegat
             return "Hold on"
         case .expanded:
             return "Drag down or tap"
+        @unknown default:
+            return "Unknown value"
```

Comment:
I'm a bit unsure about this. I feel like there should be a way to tell Swift that the enum cannot be extended anymore?

Reply:
There is, but only for Swift enums, because for ObjC, it can never know for sure (you can use funny tricks to have private enum members, e.g.).